app/scripts/views/mixins/password-mixin.js

```diff
@@ -68,33 +136,46 @@ define(function (require, exports, module) {
     },
 
     /**
-     * Set the password visibility for an element. Ensure the "show" button's
-     * state is synchronized.
+     * Make a password field's contents visible.
      *
-     * @param {String | Element} which - If a string, assumed to be a selector
-     * @param {Boolean} isVisible
+     * @param {String|Element} passwordEl
      */
-    setPasswordVisibility (which, isVisible) {
-      const $passwordEl = $(which);
+    showPassword (passwordEl) {
+      const $passwordEl = this.$(passwordEl);
+      try {
+        $passwordEl.attr('type', 'text').attr('autocomplete', 'off')
+          .attr('autocorrect', 'off').attr('autocapitalize', 'off');
+      } catch (e) {
+        // IE8 blows up when changing the type of the input field. Other
+        // browsers might too. Ignore the error.
```

Comment:
Reading farther down, I see this has been that way for a while. If not concerned, ignore.